lambda-otel4s/shared/src/main/scala/feral/lambda/otel4s/Attributes.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda.otel4s
+
+import feral.lambda.Context
+import org.typelevel.otel4s.Attribute
+import org.typelevel.otel4s.AttributeKey
+
+/**
+ * Temporary aliases for Lambda message-specific attributes in otel4s-semconv-experimental
+ */
+object LambdaMessageAttributes {
```

Comment:
Should attributes be package private?

Reply:
I think this was a shim for otel4s-semconv-experimental anyway, back when I thought this would merge before that was available, I can probably replace this

Reply:
`lambda-otel4s` still cannot depend on the `otel4s-semconv-experimental`, because semconv-experimental may receive binary breaking updates. And this will affect feral compatibility. 

So far, I came up with the following strategy:
1) Redefine required attributes in the library (like you did), ideally in a private package
2) Use `otel4s-semconv-experimental` in tests to ensure your attributes match the specification (even the experimental one). 
3) Once attributes get promoted to the `otel4s-semconv` stable artifact - replace handcrafted attributes with the stable one

Reply:
Added some specific attribute tests, a little bit boilerplatey but did catch a small issue